contracts/Conference.sol

```diff
@@ -0,0 +1,40 @@
+pragma solidity ^0.4.2;
```

Comment:
can we use the latest one?

Reply:
^0.4.2 means that any compiler above this version can be used. The latest one is 0.4.25, so there's a slight risk in using it directly, because we may not always find the latest compiler version. So it's safe to have a range of compiler versions.

Reply:
0.5.0 is about to come. Anyway, let's merge this